rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
+
+  def initialize(date)
+    @date = date
+
+    @book = book
+    book.rentals << self
+
+    @person = person
+    person.rentals << self
```

Comment:
- Well done on the rental adding itself to the @book achieving the `belongs-to` association 👏🏾 
- Also, well done on the rental adding itself to the @person achieving the  `belongs-to` association 🎆